src/sql/workbench/contrib/query/browser/gridPanel.ts

```diff
@@ -482,6 +482,7 @@ export abstract class GridTableBase<T> extends Disposable implements IView {
 		this.table.registerPlugin(new AdditionalKeyBindings());
 		this._register(this.table.onContextMenu(this.contextMenu, this));
 		this._register(this.table.onClick(this.onTableClick, this));
+		this._register(this.table.onHeaderClick(this.onHeaderClick, this));
```

Comment:
It might help adding a comment on this line of what this listener is responsible for.